applications/DeepSpeed-Chat/training/step3_rlhf_finetuning/ppo_trainer.py

```diff
@@ -143,7 +148,7 @@ def generate_experience(self, prompts, mask, step):
             'rewards': reward_score,
             'input_ids': seq,
             "attention_mask": attention_mask
-        }
+        }, generate_time
```

Comment:
Nit: Can we make `generate_time` a member variable of `DeepSpeedPPOTrainer` so that we don't add extra return values to `generate_experience()`?

This may help with keeping the API simpler for external users (without changing it) and we'll still be able to assign/source `generate_time` via `self.generate_time`.